### PR TITLE
feat: RFID and actuator implemented

### PR DESCRIPTION
**Pull request by Bosurgi**
Commits from c3591392 it was me from University computers.

### CODE REVIEW

File: scanner.py [hunk 1]
```diff
@@ -42,12 +49,21 @@ def light(color):
...
-                light(GREEN)
-            else:
-                light(RED)
-
-    except KeyboardInterrupt:
-        print("\nExiting")
+        while reading:
+            print("\nScan Card")
+            # Fetching the card UID and converting it to HEX
+            card_id = f'{reader.read_id():x}'.upper()
```
Comment (aryanprince) [inline scanner.py:55]:
```suggestion
            card_id = f'{reader.read_id():X}'
```

ez trick to make it uppercase xd

**Review by aryanprince — CHANGES_REQUESTED** [review]
Check out the comments!

File: scanner.py (hunk 1 above)
Comment (Bosurgi) [reply scanner.py:55]: Perfect!!

**Review by Bosurgi — COMMENTED** [review]
All good

**Review by aryanprince — APPROVED** [review]
Thanks, looks good to me!

**Comment by aryanprince** [comment]
@Bosurgi only read the PR description now lol

**Pull request merged**